src/ore/src/collections.rs

```diff
@@ -87,6 +87,18 @@ pub trait AssociativeExt<K, V> {
     fn unwrap_insert(&mut self, k: K, v: V) {
         self.expect_insert(k, v, "called `unwrap_insert` for an already-existing key")
     }
+
+    /// Removes a key, panicking with
```

Comment:
The wrapping here is weird

Reply:
I agree, but I was trying to match the formatting of `expect_insert` and `unwrap_insert` above which has similarly weird formatting.

Reply:
Lol! I didn't notice that 🙃